### ISSUE TITLE
Add --tr-testrun-refs option

### ISSUE THREAD

**Issue by mickyJNST** [allankp/pytest-testrail#155]
**Is your feature request related to a problem? Please describe.**
TestRail allows for integration with Jira via the `refs` parameter in test runs. **pytest-testrail** doesn't currently support setting the `refs` param (which is just a comma delimited string).

**Describe the solution you'd like**
Add `--tr-testrun-refs` option to existing list of **pytest-testrail** options

**Describe alternatives you've considered**
To work around this limitation we are first creating the test run (including the `refs` parameter) in TestRail directly via the TestRail API (before invoking our pytest-based tests), and then setting the **pytest-testrail** `--tr-run-id` option using the test run id we get from the TestRail API.

**Comment by voloxastik** [comment]
Hi @mickyJNST 
Could you explain more about your request?
Do you mean to create a new test run with test cases that contain specified refs from option --tr-testrun-refs?

**Comment by voloxastik** [comment]
As workaround, you can create directly necessary  testrun and then run pytest with option `--tr-run-id XXXXX`

**Comment by mickyJNST** [comment]
> As a workaround, you can create necessary test run directly and then run pytest with option `--tr-run-id XXXXX`

Yeah - tried this but ran into other issues (as described in the **Describe alternatives you've considered** section of original post)

**Comment by voloxastik** [comment]
Your testrail.cfg need to be seems:

> [API]
> url = xxx
> email = xxx
> password = xxx
> [TESTRUN]
> run_id = 999
> assignedto_id = 111
> project_id = 83

Then you can run your tests using following command:
 `pytest /tests/my_tests.py --testrail --tr-testrun-suite-id=24728 --tr-run-id=108689 --tr-config=/path/to/testrail.cfg`

**Comment by voloxastik** [comment]
> > As a workaround, you can create necessary test run directly and then run pytest with option `--tr-run-id XXXXX`
> 
> Yeah - tried this but ran into other issues (as described in the **Describe alternatives you've considered** section of original post)

Your testrail.cfg should look like this:

[API]
url = xxx
email = xxx
password = xxx
[TESTRUN]
assignedto_id = 111
project_id = 83

Then you can run your tests using following command:
pytest /tests/my_tests.py --testrail --tr-run-id=XXXXXX
where XXXXXX is the ID of your test run created earlier

**Comment by mickyJNST** [comment]
Thanks for the explanation @voloxastik. Unfortunately your proposed solution doesn't fit with how we have our workflow set up as we're trying to do everything via the testrail.cfg file...and we don't want to have to do a combination of file and pytest command line options. In any case, we have a very roundabout workaround in place that will do for now, but would like to see support for the test run `refs` option added (for both the config file and command line).

**Comment by voloxastik** [comment]
@mickyJNST
Could you explain more about your request?
Do you mean to create a new test run with test cases that contain specified refs from option --tr-testrun-refs?

**Comment by mickyJNST** [comment]
@voloxastik I want to associate the test run with Jira tickets. This is supported by the TestRail API via the `refs` parameter. Hopefully this clarifies what I am getting at:

![pytest-testrail-155](https://user-images.githubusercontent.com/85323292/153537355-2835348a-b05d-444c-8133-346e590cb493.png)

**Comment by voloxastik** [comment]
@mickyJNST 
FYI. The plugin does not use information about testrun_id from the config file
`run_id=config.getoption('--tr-run-id')`
only from run command
but it's easy to fix
I will do a PR for it.
